tools/internal_ci/linux/grpc_full_performance_master.sh

```diff
@@ -45,15 +45,6 @@ tools/run_tests/run_performance_tests.py \
     || EXIT_CODE=1
 
 # prevent pushing leftover build files to remote hosts in the next step.
-git clean -fdxq -e reports
-
-# selected scenarios on Windows
-tools/run_tests/run_performance_tests.py \
-    -l csharp \
-    --category scalable \
-    --remote_worker_host grpc-kokoro-performance-windows1 grpc-kokoro-performance-windows2 \
```

Comment:
Is there a plan to fix the Windows benchmarks and add them back into the suite? What's causing them to be permanently broken?

Reply:
(the VMs are out of date and refreshing them is hard). considering that our OSS benchmarks are not in good shape in general and we barely have enough cycles to keep our linux benchmarks running and to give at least somewhat reasonable outputs, I think it's the best to forget about windows benchmarking for now
We can always add windows back once we do some benchmark infrastructure revamp to make the benchmarks more reliable and easier to maintain. Yeah it's not ideal, but I think this is the right call.